tests/SigningTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use AWS\CRT\Auth\SignatureType;
+use AWS\CRT\Auth\SigningAlgorithm;
+use AWS\CRT\Auth\SigningConfigAWS;
+use AWS\CRT\Auth\Signing;
+use AWS\CRT\Auth\Signable;
+use AWS\CRT\Auth\StaticCredentialsProvider;
+use AWS\CRT\HTTP\Request;
+
+require_once('common.inc');
+
+final class SigningTest extends CrtTestCase {
+
+    public function testConfigAWSLifetime() {
```

Comment:
Not really anything to note here since these all seem like test that don't reuse a lot of code, but if you do need to test the same code against multiple inputs, you can use a dataprovider.